src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -114,15 +117,28 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
     StringBuilder builder = new StringBuilder();
     builder.append("{");
 
-    String userEmail = "unknown";
-    BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
-    getGitUser.run();
-    if (getGitUser.exitValue() == Command.SUCCESS) {
-      userEmail = getGitUser.getOutput().trim();
+    String userId = "unknown";
+    BasicCommand getUserId = null;
+    if (GatorGradlePlugin.OS.equals(GatorGradlePlugin.WINDOWS)) {
+      getUserId = new BasicCommand(
+          "sh", "/C", GatorGradleConfig.get().getIdCommand());
+    } else {
+      getUserId = new BasicCommand(
+          "sh", "-c", GatorGradleConfig.get().getIdCommand());
+    }
+    getUserId.run();
+    if (getUserId.exitValue() == Command.SUCCESS) {
+      userId = getUserId.getOutput().trim();
+    } else {
+      try {
```

Comment:
We should log something (maybe an error if that isn't fatal) here to indicate that we're falling back on the hostname. I think it's probably okay to still send it but at least visibly warn the user.

Actually, in thinking about it, I think it might be best to fail if the id command doesn't run, because we don't want to pollute the database. What do you think?

Reply:
I agree. It could also be hard to query too later on. I will make the changes now.